analyses/cell-type-nonETP-ALL-03/scripts/00-01_processing_rds.R

```diff
@@ -43,7 +43,7 @@ run_sam <- function(ind.sample, ind.library){
   final.obj[["ADT"]]@data <- logcounts(altExp(sce, "adt"))
   final.obj[["ADT"]] <- AddMetaData(final.obj[["ADT"]], as.data.frame(rowData(altExp(sce, "adt"))))
   
-  saveRDS(final.obj, file.path(out_loc,"scratch",paste0(ind.sample,".rds")))
+  saveRDS(final.obj, file.path(out_loc,"results/rds",paste0(ind.sample,".rds")))
```

Comment:
Yes, for the ETP-ALL samples, Sample SCPCS000066 has two libraries, and I actually merge the two rds file without doing any batch effect correction. I write another function of `run_multisam` for those with multiple libraries. It requires a little bit of manual code to provide input to that function, but it works perfectly fine for ETP-ALL samples, since there is only one case. 
<img width="1252" alt="Screenshot 2024-10-03 at 11 12 17 AM" src="https://github.com/user-attachments/assets/f9f940e3-146e-4f5d-8780-4d2ffb044892">

That's the reason why I thought saving as sampleID making more sense, because technically we should merge multiple libraries together if they belong to the same sample? Isn't this will make the analysis more powerful, since there are more cells added?

Reply:
While it seems like the batch effect with that case is not bad, I would still lean toward leaving the two libraries separate. Having more cells may not always result in better results (especially if there were a batch effect), but that is largely an empirical question; I think we would want to evaluate what the tradeoffs are for different parts of the analysis. 

To me, keeping the libraries separate seems like the more conservative (and simpler) approach, so it is the one I would start with.

Reply:
Sure, I will remove the `run_multisam` function from the ETP-ALL module script.